### ISSUE TITLE
Do not force to use `yarn`

### ISSUE THREAD

**Issue by deadcoder0904** [react-community/create-react-native-app#324]
Allow users to run the command rather than forcing 😃 

I think people can type `npm i` or `yarn` after doing `create-react-native-app myApp`

My main concern is due to I want to use [pnpm](https://github.com/pnpm/pnpm) which is fast & saves a lotta disk space than both `npm` & `yarn` 😉 

As a guy doing a lotta projects in `react-native` I don't think I need to waste space for more `node_modules`  😜

**Comment by brentvatne** [comment]
> As a guy doing a lotta projects in react-native I don't think I need to waste space for more node_modules 😜

lol

> I think people can type npm i or yarn after doing create-react-native-app myApp

99% of the time you will be doing this, and this is the behavior of create-react-app, so we aren't going to change this. if you want to add a flag to disable it, then that would be fine! a PR is welcome for that.

**Comment by brentvatne** [comment]
I forgot about this commit ;) https://github.com/react-community/create-react-native-app/commit/1609cef0b5e4429631516b3edbed43248e32aab4

you can specify `--package-manager` when running `create-react-native-app` if you want! will cut a release with this shortly

**Comment by deadcoder0904** [comment]
Thanks !!  I knew there must be some flag but couldn't figure it out 😂

**Comment by deadcoder0904** [comment]
`pnpm` also supported #412

**Comment by vjpr** [comment]
I see this message following an install with pnpm.

> *******************************************************************************
> ERROR: npm 5 is not supported yet
> *******************************************************************************
>
> It looks like you're using npm 5 which was recently released.
>
> Create React Native App doesn't work with npm 5 yet, unfortunately. We
> recommend using npm 4 or yarn until some bugs are resolved.
>
> You can follow the known issues with npm 5 at:
> https://github.com/npm/npm/issues/16991
>
> *******************************************************************************

**Comment by deadcoder0904** [comment]
@vjpr Got busy with other things so couldn't send a PR. Still I tried using `pnpm` with `react-native init` & it doesn't work because of [this issue](https://github.com/pnpm/pnpm/issues/940) so I don't know if its worth the trouble yet. I'm currently using `npm` & `yarn` bcz sometimes `pnpm` has issues as it isn't matured yet for some of my usecases. I have filed an [issue](https://github.com/facebook/metro/issues/119) there so if it works out. I'll make sure to send a PR.
